client/DigiDoc.h

```diff
@@ -78,7 +78,8 @@ class DigiDocSignature
 		Warning,
 		Test,
 		Invalid,
-		Unknown
+		Unknown,
+		NonQSCD
```

Comment:
The order is important, NonQSCD masks Invalid and Test levels. It should be before or after Warning level